js/dataManager.js

```diff
@@ -16,7 +12,7 @@ class DataManager {
       6: 0, 7: 0, 8: 0, 9: 0, 10: 0
     };
 
-    return this.data.clues.reduce((acc, clue) => {
+    return data.clues.reduce((acc, clue) => {
```

Comment:
What is the purpose of not including this? Does this mean that we do not need the property at all in the constructor?